WalletWasabi.Gui/Controls/WalletExplorer/CoinJoinTabViewModel.cs

```diff
@@ -264,20 +264,26 @@ private async Task DoEnqueueAsync(IEnumerable<CoinViewModel> selectedCoins)
 
 		private void UpdateStates()
 		{
-			AmountQueued = Global.ChaumianClient.State.SumAllQueuedCoinAmounts();
+			var chaumianClient = Global?.ChaumianClient;
+			if (chaumianClient == null)
+			{
+				return;
```

Comment:
Until now here we had an exception here. Now we only have a silent return. Is it important to throw the exception to have an error message, isn't it?

Reply:
I agree we should not throw anything during the exit procedure.